### DISCUSSION TITLE
Can't I use this document as proof of my current degree?

### DISCUSSION THREAD

**Discussion by Manju-S-6271** [community/community#113673] [GitHub Education] [answered]
### Hi everyone,

I want to use GitHub Education, but everything I send them is rejected no matter how many times I send it.
Why is this?

By the way, the school name on this and the school name on GitHub are slightly different. The official English certificate was prepared by the principal of my school, but there is a discrepancy.

Also, the school email address is @~~~~.onmicrosoft.com.

My certificate (some personal information is hidden by me) are as follows:

![IMG_20240322_0002_edited](https://github.com/community/community/assets/153196799/6a85ff24-97c3-4a95-8980-a9a5f331fc94)

**Comment by samus-aran** [comment]
Hi @Manju-S-6271 👋 

I know you've seen the latest **[Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352)** but I'll highlight some areas:

- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

> You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this.
> 
> Accepted types of evidence include a link to a publicly-accessible page on the school website or an affidavit from your school on school letterhead, detailing their current academic email issuing policy, or listing their academic domain(s), as it is relevant in your case. 👈

- [I need to translate my evidence. What can I do?](https://github.com/orgs/community/discussions/111352#translate-my-evidence)
> Ensure your image is legible. If you are unable to read the text on the image itself then the verification process may not be able to either.

- [The image you selected does not appear to contain your school name.](https://github.com/orgs/community/discussions/111352#image-does-not-appear-to-contain-your-school-name)

> You may include multiple documents in your image. We look at all of the documents you submit collectively.
> Logos are not sufficient.

Lastly,  **submit a live photo of their evidence rather than an upload.** You can include handwritten translates next to your pieces of evidence.

**Reply by Manju-S-6271** [reply, marked as answer]
Thank you @samus-aran for your response.
I solved the problem by making a paper with large letters (with the name of the school, my name and the date of graduation).
Have a nice day.
